test/stdlib/Result.swift

```diff
@@ -0,0 +1,178 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import Swift
+
+let ResultTests = TestSuite("Result")
+
+fileprivate enum Err: Error, Equatable {
+  case err
+  case derr
+}
+
+fileprivate let string = "string"
+
+fileprivate extension Result {
+  var value: Value? {
+    switch self {
+    case let .value(value):
+      return value
+    case .error:
+      return nil
+    }
+    
+    var error: Error? {
```

Comment:
This is nested inside `var value: Value?`, which is why you aren't able to call `result.error`. IMO, it's better to just remove this extension and remove those tests.

Reply:
Thanks, fixed in [7fb5905](https://github.com/apple/swift/pull/20958/commits/7fb5905f5f5b9076769508e35882bcd1380a755e). I'd like to leave the tests for now, as they're the only ones testing the unwrapping. I can remove the property extension if you think that's important.